docs/sources/drill-down-metrics/index.md

```diff
@@ -0,0 +1,62 @@
+---
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Drill down your metrics
+weight: 30
+refs:
+  get-started:
+    - pattern: /docs/grafana/
+      destination: https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/simplified-exploration/metrics/get-started/
+    - pattern: /docs/grafana-cloud/
+      destination: https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration/metrics/get-started/
+---
+
+# Drill down your metrics
+
+Drill down into your metrics to gain insight into your data without writing a query. First, filter metrics to appear in the **Metrics Drilldown** dashboard. Then, perform an in-depth analysis of selected metrics.
```

Comment:
It feels like the word "dashboard" might be misleading.

Perhaps something more along these lines?

```suggestion
Drill down into your metrics to gain insight into your data without writing a query. First, filter the metrics surfaced by **Metrics Drilldown**. Then, select a metric to perform in-depth analysis.
```